### PR TITLE
Em 21

### PR DESCRIPTION
**Pull request by LuzLimon**

### CODE REVIEW

**Review by MiguelAngelSF — DISMISSED** [review]

**Review by Navi02643 — DISMISSED** [review]

**Review by MiguelAngelSF — DISMISSED** [review]

File: src/components/Login/Login.module.css [hunk 1]
```diff
@@ -1,141 +1,212 @@
 .container {
```
Comment (lalodiazdiaz) [inline src/components/Login/Login.module.css:1]: image

identation

File: src/components/Navbar/Navbar.module.css [hunk 2]
```diff
@@ -32,8 +32,7 @@
   padding: 5px;
   border-radius: 5px;
   width: 130px;
-  background: #66d086;
-  border: 0;
+  background: rgba(13, 177, 108, 0.79);   border: 1px solid black;
```
Comment (lalodiazdiaz) [inline src/components/Navbar/Navbar.module.css]: image

add space

File: src/services/Login/LoginService.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const URL = 'https://emociones-be-production.up.railway.app/api/';
+
+const getLogin = async (body) => {
+	const resp = await axios.post(URL, body);
+	console.log(resp);
+	// if (data.data.error === false) {
```
Comment (lalodiazdiaz) [inline src/services/Login/LoginService.jsx]: image

remove comments

File: src/components/Login/Login.module.css (hunk 1 above)
Comment (lalodiazdiaz) [inline src/components/Login/Login.module.css:1]: check identation

File: src/components/Login/Login.jsx [hunk 4]
```diff
@@ -1,83 +1,98 @@
 import React, { useState } from 'react';
```
Comment (lalodiazdiaz) [inline src/components/Login/Login.jsx:1]: Check identation

**Review by MiguelAngelSF — APPROVED** [review]

**Review by lalodiazdiaz — APPROVED** [review]

**Pull request merged**